pyproject.toml

```diff
@@ -1,20 +1,3 @@
 [tool.black]
 line-length = 79
-target-version = ['py39']
-include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs         # black-default
-    | \.git
-    | \.hg
-    | \.mypy_cache
-    | \.tox
-    | \.venv
-    | _build
-    | buck-out
-    | build
-    | dist
-  )/
-)
-'''
```

Comment:
Is the removal of `include` and `exclude` intentional? If so, fine for me.

Reply:
Yes, I removed this additional config in [`aeabb02`](https://github.com/unitaryfund/mitiq/pull/1504/commits/aeabb02a9caf71dac27be0f28ad82f548691f589). The motivation for doing so is to ensure we are using whatever `include`/`exclude` patterns black recommends. The above lists have changed slightly in black since we added this code, so I think it's best if we just default to what they recommend.